tool.go

```diff
@@ -266,6 +258,13 @@ func main() {
 		Use:   "run [gofiles...] [arguments...]",
 		Short: "compile and run Go program",
 	}
+	cmdRun.Flags().AddFlag(flagVerbose)
```

Comment:
Create multiple flag sets and add them. Flag sets consisting of only a single flag are totally fine. That way you can avoid `Lookup` completely.